### PR TITLE
Implemented boxing wrappers for primitive sets. #1408.

### PR DESCRIPTION
**Pull request by goldbal330**

### CODE REVIEW

**Comment by donraab** [comment]
@goldbal330 I think you meant this to be "Implemented boxing wrappers for primitive sets."

**Review by donraab — CHANGES_REQUESTED** [review]
Thanks for the contribution @goldbal330 ! A few changes requested inline.

File: eclipse-collections-code-generator/src/main/resources/impl/set/mutable/boxedPrimitiveSet.stg [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import org.eclipse.collections.impl.set.mutable.AbstractMutableSet;
+
+/**
+ * A boxed view of a {@link Mutable<name>Set}.
+ *
+ * This is for scenarios where optimization is not a consideration and therefore no further methods will be optimized.
+ * \<p>
+ * This file was automatically generated from template file boxedPrimitiveSet.stg.
+ *
+ * @since 12.0.1
```
[the remark this answers is not in the document]
Comment (goldbal330) [reply eclipse-collections-code-generator/src/main/resources/impl/set/mutable/boxedPrimitiveSet.stg]: Fixed

File: .idea/jarRepositories.xml
[the remark this answers is not in the document]
Comment (goldbal330) [reply .idea/jarRepositories.xml:1]: Removed.

**Review by donraab — APPROVED** [review]
LGTM. Thanks for the contribution @goldbal330 !

**Pull request merged**